packages/core/lib/debug/interpreter.js

```diff
@@ -738,6 +750,11 @@ class DebugInterpreter {
   updatePrintouts(userArgs, selections, printOuts) {
     let tempPrintouts = new Set();
     for (let argument of userArgs) {
+      // skip line number options
+      if (((argument[0] === "-") || (argument[0] === "+")) &&
+        (!isNaN(argument.slice(1))))
```

Comment:
Hm, using `isNaN` like this is a bit opaque.  At least to me; maybe it's common elsewhere, I dunno.  I'd suggest changing it to `isNaN(Number(...))` for clarity, and maybe adding a comment.  Maybe that's overkill, I dunno, but I'd suggest clarifying it somehow at least.

Reply:
The code is deleted, see the comment above.